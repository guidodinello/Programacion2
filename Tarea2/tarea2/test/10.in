# Implementar insertarSegmentoDespues
insertarSegmentoDespues 0
imprimirCadena
insertarSegmentoDespues 1 (5,3.4)
imprimirCadena
finalCadena
insertarSegmentoDespues 0
imprimirCadena
removerDeCadena
inicioCadena
insertarSegmentoDespues 3 (5,3.4) (8,9.4) (9,2.6)
imprimirCadena
inicioCadena
insertarSegmentoDespues 0
imprimirCadena
insertarSegmentoDespues 2 (6,5.2) (7,0.9)
imprimirCadena
Fin
