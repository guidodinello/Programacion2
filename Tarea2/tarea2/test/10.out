1>#  Implementar insertarSegmentoDespues.
2>Segmento insertado después de loc.
3>
4>Segmento insertado después de loc.
5>(5,3.40)
6>loc al final.
7>Segmento insertado después de loc.
8>(5,3.40)
9>Removido.
10>loc al inicio.
11>Segmento insertado después de loc.
12>(5,3.40)(8,9.40)(9,2.60)
13>loc al inicio.
14>Segmento insertado después de loc.
15>(5,3.40)(8,9.40)(9,2.60)
16>Segmento insertado después de loc.
17>(5,3.40)(6,5.20)(7,0.90)(8,9.40)(9,2.60)
18>Fin.
